ML/CheckDistr.py

```diff
@@ -23,8 +27,10 @@ def main():
 
     print('Loading data files: ...', end='\r')
     DfList = []
+    inTreeName = inputCfg['input']['treename']
```

Comment:
You could add also a `inDirName` variable read from the config file so that one can have either `""` or a directory name